.changeset/chocolate-cake.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
BTW, if you prefix the changeset file name w/`techdocs-`, we might be able to avoid the @backstage/reviewers approval.

Reply:
Fixed, thank you!